docs/general/information-schema.rst

```diff
@@ -952,6 +953,36 @@ See :ref:`administration-fdw`.
      - ``TEXT``
      - Name of the foreign server
 
+.. _foreign_table_options:
+
+``foreign_table_options``
+-------------------------
+
+Lists options for foreign tables created using :ref:`ref-create-foreign-table`.
+See :ref:`administration-fdw`.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Column Name
+     - Return Type
+     - Description
+   * - ``foreign_table_catalog``
+     - ``TEXT``
+     - Name of the database that contains the foreign table. Always ``crate``.
```

Comment:
```suggestion
     - Name of the database that contains the foreign table. Value is always ``crate``.
```

Reply:
I'd rather leave it for consistency with the others.